src/plots/cartesian/graph_interact.js

```diff
@@ -60,7 +60,7 @@ fx.supplyLayoutDefaults = function(layoutIn, layoutOut, fullData) {
     coerce('dragmode');
 
     var hovermodeDflt;
-    if(layoutOut._hasCartesian) {
+    if(layoutOut._hasCartesian && !layoutOut._hasPie) {
```

Comment:
Easy. Done in https://github.com/plotly/plotly.js/pull/491/commits/f476e05b09a7a9af1bb2689fb10f1121c9c1c72b